Release 6.0alpha6 (#141)
JabRef/blog.jabref.org: _posts/2026-05-14-JabRef6-0-alpha6.md

---
title: 🌞 JabRef 6.0-alpha.6 Release and three GSOC projects 🌞
tags: [release]
---

It's just May and we again have a full changelog since the last release.

> And JabRef once again got accepted for this year's [Google Summer of Code (GSOC)](https://summerofcode.withgoogle.com/programs/2026/organizations/jabref-ev)

Reviewer: ```suggestion
And JabRef once again got accepted for this year's [Google Summer of Code (GSOC)](https://summerofcode.withgoogle.com/programs/2026/organizations/jabref-ev)!
```

We are happy to announce the release of **JabRef 6.0 Alpha 6**, the next step towards the stable version of JabRef 6.0.

> It's formally an alpha release because we have a couple of things left in our backlog for a Beta version. But nontheless this version includes plenty of fixes and improvemen and thus we invite everyone to try out this version and report any encountered issues to make JabRef even more stable.

Reviewer 2: ```suggestion
It's formally an alpha release because we have a couple of things left in our backlog for a Beta version. But nonetheless this version includes plenty of fixes and improvements and thus we invite everyone to try out this version and report any encountered issues to make JabRef even more stable.
```

## Release Highlights

> - Improved theming in JabRef - Big thanks to @Maran32!

Reviewer 2: 🧐

> - Improved theming in JabRef - Big thanks to @Maran32!

Reviewer: Maran 23 instead of 32

Author: I put it at the end

- Revamped notifications and pop-up toast into a single center info center.

> - Improved Usability e.g. for citation relations

Reviewer: ```suggestion
- Improved usability e.g. for citation relations
```
Also I think this needs to be more descriptive

- Improved generation and and handling of inserted citations in LibreOffice

> 

Reviewer 3: You can now paste publisher-specific search queries directly into the web search bar
or 
Web search now accepts publisher-specific query syntax.

Author: done

> Despite these, there were plenty of bug fixes across all parts of JabRef. Check out our Changelog for all changes and fixes.

Reviewer 2: Put a link to changelog?

Reviewer 4: `- Improved generation and and handling of inserted citations in LibreOffice`, double `and`

Author: Ah yeah I missed the link

> Despite these, there were plenty of bug fixes across all parts of JabRef. Check out our Changelog for all changes and fixes.

Reviewer: ```suggestion
Apart from these, there were plenty of bug fixes across all parts of JabRef. Check out our Changelog for all changes and fixes.
```

## Known issues

> - Custom themes may not work in the current Version, however we are working on making them compatible with the new theming system.

Reviewer 2: `Version` or `version`?

  
## Google Summer of Code

JabRef is participating in GSOC this summer with three projects:

- Zeyad Abdelfattah will be working on "Improved handling of older documents with OCR and AI-powered tools."
- Hancong Zhang will work on "Improved LibreOffice-JabRef integration" with one particular aspect of compatibility with other reference managers.

> - Wanling FU  will work on improving startup times for our CLI-based tool JabKit by leveraging the power of Hashtag GraalVM.

Reviewer 2: ```suggestion
- Wanling FU will work on improving startup times for our CLI-based tool JabKit by leveraging the power of Hashtag GraalVM.
```

Stay tuned for more project updates during the summer.

### Special Thanks

This time a special thanks to [Terry Bollinger](https://terrybollinger.com/) for sponsoring JabRef!

> A special thanks to [Marius](https://github.com/Maran23) who fixes longstanding bugs in the UI framework JavaFX that surfaced in JabRef.

Reviewer: Since we are doing it here, we should add github links for the other people mentioned as well

Author: will do

> It’s people like those who help keep this software going.

Reviewer: ```suggestion
It’s people like these who help keep this software going.
```

We thank all the external contributors who contributed code to this JabRef release.

> TODO: Add contributors from script

Reviewer: Also TODO: add a mention our SLR team.

Reviewer: Faneesh Juneja (@faneeshh) and Loay Tarek Mostafa (@LoayTarek5).
(Their reviews also helped us get a couple of bugfixes and features merged.)

It's formally an alpha release because we have a couple of things left in our backlog for a Beta version. But nonetheless this version includes plenty of fixes and improvements and thus we invite everyone to try out this version and report any encountered issues to make JabRef even more stable.

## Release Highlights

- Improved theming in JabRef
- Revamped notifications and pop-up toast into a single center info center.
- Improved citation count and citation relations feature, e.g you can now chose between different fetchers for updating the citation count. Drag and drop of entries from the citation relations tab is also now possible.
- Improved generation and and handling of inserted citations in LibreOffice
- Web search now accepts publisher-specific query syntax as well

Apart from these, there were plenty of bug fixes across all parts of JabRef. Check out our [Changelog](https://github.com/JabRef/jabref/blob/main/CHANGELOG.md) for all changes and fixes.

TODO: Replace changelog link with correct version

![Fetcher selection in the citations count field](../img/citationscount.png)

> 

Reviewer 5: You can now link the unlinked files to existing related entries, not just add them as new entries.

<img width="1017" height="902" alt="Image" src="https://github.com/user-attachments/assets/1b0bd4c2-3e6f-4d77-83ee-765364df05f5" />

You can now clear a group with just one click.
